docs/concepts.rst

```diff
@@ -158,6 +158,96 @@ Example DAG with functional abstraction
 
 .. _concepts:dagruns:
 
+executor_config
```

Comment:
concepts.rst should have only the core concepts. Additional information specific to a specific topic should be included in the topic article.  For example, queues are unique to CeleryExecutor, so they are described in [CeleryExecutor](https://airflow.readthedocs.io/en/latest/executor/celery.html#queues).